docs/content/get-started/installation/controller.md

```diff
@@ -162,16 +162,32 @@ into your cluster.
 
    **Note**: Please make sure that the flag `web.enable-remote-write-receiver`
    is enabled on your existing Prometheus instance as it is required by the
-   Controller.
+   Aperture Controller.
 
-5. If you want to modify the default parameters, you can create or update the
-   `values.yaml` file and pass it with `helm upgrade`:
+5. If you want to modify the default parameters or the Aperture Controller
```

Comment:
Should this section be repeated 3 times? Can we instead have it at a common place?